### PR TITLE
Update SDK endpoints

### PR DESCRIPTION
**Pull request by smithki**
## Description

- Updates endpoints to use new schemas and 0x.js v1
- Depends on #42

### CODE REVIEW

File: test/lib/mockRequests.ts [hunk 1]
```diff
@@ -82,36 +84,56 @@ export function mockRequests() {
...
+      senderAddress: '0x0000000000000000000000000000000000000000',
+      makerAddress: 'SET',
+      takerAddress: '0x0000000000000000000000000000000000000000',
+      makerFee: new BigNumber(0),
+      takerFee: new BigNumber(0),
+      makerAssetAmount: new BigNumber(10).pow(14),
+      takerAssetAmount: new BigNumber(10).pow(14),
+      makerAssetData: '0xf47261b0000000000000000000000000d0a1e359811322d97991e03f863a0c30c2cf029c',
+      takerAssetData: '0xf47261b00000000000000000000000006ff6c0ff1d68b964901f986d4c9fa3ac68346570',
+      salt: '1536516727297',
```
Comment (agustingabiola) [inline test/lib/mockRequests.ts]: `salt` should be a BigNumber

File: test/lib/mockRequests.ts [hunk 2]
```diff
@@ -82,36 +84,56 @@ export function mockRequests() {
...
-      takerTokenAddress: '0x34d402f14d58e001d8efbe6585051bf9706aa064',
-      takerTokenAmount: new BigNumber(10).pow(14)
+      senderAddress: '0x0000000000000000000000000000000000000000',
+      makerAddress: 'SET',
+      takerAddress: '0x0000000000000000000000000000000000000000',
+      makerFee: new BigNumber(0),
+      takerFee: new BigNumber(0),
+      makerAssetAmount: new BigNumber(10).pow(14),
+      takerAssetAmount: new BigNumber(10).pow(14),
+      makerAssetData: '0xf47261b0000000000000000000000000d0a1e359811322d97991e03f863a0c30c2cf029c',
```
Comment (agustingabiola) [inline test/lib/mockRequests.ts]: You should use asset data encoding for this.

File: test/lib/mockRequests.ts [hunk 3]
```diff
@@ -82,36 +84,56 @@ export function mockRequests() {
...
+      takerAssetData: '0xf47261b00000000000000000000000006ff6c0ff1d68b964901f986d4c9fa3ac68346570',
```
Comment (agustingabiola) [inline test/lib/mockRequests.ts]: ditto

File: test/lib/mockRequests.ts [hunk 4]
```diff
@@ -82,36 +84,56 @@ export function mockRequests() {
...
-      feeRecipient: '0xa258b39954cef5cb142fd567a46cddb31a670124',
-      makerFee: '0',
-      makerTokenAddress: '0x48bacb9266a570d521063ef5dd96e61686dbe788',
-      makerTokenAmount: new BigNumber(10).pow(14),
-      salt: '50383144753900994556920531501252803163709013513245472190540889193319785504083',
-      taker: '0x0000000000000000000000000000000000000000',
-      takerFee: '0',
-      takerTokenAddress: '0x34d402f14d58e001d8efbe6585051bf9706aa064',
-      takerTokenAmount: new BigNumber(10).pow(14)
+      senderAddress: '0x0000000000000000000000000000000000000000',
```
Comment (agustingabiola) [inline test/lib/mockRequests.ts]: Create const with NULL address `static NULL_ADDRESS = '0x0000000000000000000000000000000000000000';`

File: test/lib/mockRequests.ts [hunk 5]
```diff
@@ -82,36 +84,56 @@ export function mockRequests() {
...
+      makerAddress: 'SET',
```
Comment (agustingabiola) [inline test/lib/mockRequests.ts]: I think they don't use SET anymore, @cavanmflynn ?

File: test/lib/mockRequests.ts [hunk 6]
```diff
@@ -82,36 +84,56 @@ export function mockRequests() {
...
+      makerFee: new BigNumber(0),
+      takerFee: new BigNumber(0),
+      makerAssetAmount: new BigNumber(10).pow(14),
+      takerAssetAmount: new BigNumber(10).pow(14),
+      makerAssetData: '0xf47261b0000000000000000000000000d0a1e359811322d97991e03f863a0c30c2cf029c',
+      takerAssetData: '0xf47261b00000000000000000000000006ff6c0ff1d68b964901f986d4c9fa3ac68346570',
+      salt: '1536516727297',
+      exchangeAddress: '0x35dd2932454449b14cee11a94d3674a936d5d7b2',
+      feeRecipientAddress: '0xa258b39954cef5cb142fd567a46cddb31a670124',
+      expirationTimeSeconds: '1536559895',
```
Comment (agustingabiola) [inline test/lib/mockRequests.ts]: Also a BigNumber

File: test/lib/mockRequests.ts [hunk 7]
```diff
@@ -82,36 +84,56 @@ export function mockRequests() {
...
+      signature: 'SET',
```
Comment (agustingabiola) [inline test/lib/mockRequests.ts]: No longer a SET

File: test/lib/mockRequests.ts (hunk 6 above)
Comment (aacotroneo) [reply test/lib/mockRequests.ts]: For the record.. this were not BigNumber's as this are the mocks returned by the API... the API call returns strings.. it's the SDK's responsibility to transform this into a BigNumber somwhere else

**Pull request merged**